schema/fda_drugs_schema.mcf

```diff
@@ -0,0 +1,168 @@
+Node: dcid:Drug
+name: "Drug"
+typeOf: schema:Class
+subClassOf: schema:ChemicalCompound
+description: "A chemical or biologic substance, used as a medical therapy, that has a physiological effect on an organism. Here the term drug is used interchangeably with the term medicine although clinical knowledge make a clear difference between them."
```

Comment:
This one already exists. Remove this and any other node that already exists in the schema?

Reply:
From a quick skim, others include: sponsor, manufacturer

Reply:
I think this is fixed. @clincoln8  I will resolve this.